src/Microsoft.Health.Fhir.Api/Registration/FhirServerApplicationBuilderExtensions.cs

```diff
@@ -23,10 +24,21 @@ public static class FhirServerApplicationBuilderExtensions
         /// <param name="app">The application builder instance.</param>
         /// <returns>THe application builder instance.</returns>
         public static IApplicationBuilder UseFhirServer(this IApplicationBuilder app)
+        {
+            return app.UseFhirServer(null);
+        }
+
+        /// <summary>
+        /// Adds FHIR server functionality to the pipeline with health check filter.
+        /// </summary>
+        /// <param name="app">The application builder instance.</param>
+        /// <param name="predicate">The predicate used to filter health check services.</param>
+        /// <returns>THe application builder instance.</returns>
+        public static IApplicationBuilder UseFhirServer(this IApplicationBuilder app, Func<HealthCheckRegistration, bool> predicate)
```

Comment:
Could these two methods just be collapsed with a default value?
```suggestion
        public static IApplicationBuilder UseFhirServer(this IApplicationBuilder app, Func<HealthCheckRegistration, bool> predicate = null)
```

Reply:
yes, it could. Already changed to that format.